src/scripts/utils.py

```diff
@@ -394,13 +393,25 @@ def jinja_sparql(
     command_str = f'robot query --input {onto_path} --query {instantiated_query_path} {results_path}'
 
     # Cache and run
+    # - If return from cache, empties existing cache before running
     os.makedirs(results_dirpath, exist_ok=True)
     if not (os.path.exists(results_path) and use_cache):
+        os.remove(instantiated_query_path)
+        os.remove(command_str)
+        os.remove(results_path)
         with open(instantiated_query_path, 'w') as f:
             f.write(instantiated_str)
         with open(command_save_path, 'w') as f:
             f.write(command_str)
-        result = subprocess.run(command_str.split())
+        try:
+            result = subprocess.run(command_str.split())
+        except FileNotFoundError as e:
+            if 'robot' in str(e):
+                # joeflack4 2023/04/25: Suddenly my PATH is wrong. Could be virtualenvwrapper+PyCharm issue.
+                command_str = command_str.replace('robot query', '/usr/local/bin/robot query')
```

Comment:
There's a strange issue on my end that I could not quickly solve. Without this change, I was not able to run things, so leaving it this way for now but we can discuss if you want. This only happens when running in PyCharm (for debugging). When I look at my `PATH` in my terminal with my virtualenv active, it is correct.
>                 `# joeflack4 2023/04/25: Suddenly my PATH is wrong. Could be virtualenvwrapper+PyCharm issue.`